elements/rh-footer/rh-footer-social-link.ts

```diff
@@ -29,11 +29,16 @@ export class RhFooterSocialLink extends LitElement {
       const newDiv = oldDiv.cloneNode(true) as Element;
       // remove the _rendered content
       newDiv.querySelectorAll('[_rendered]').forEach(i => i.remove());
-      // NB: icons are restricted to fab set, so as not to require a minor release
-      // rh-icon is slated to deal with this problem in-house
-      newDiv.innerHTML = `<pf-icon icon="${this.icon ?? ''}"
-                                   set="fab"
-                                   loading="eager">${newDiv.innerHTML}</pf-icon>`;
+      /* Temporary fix for x-twitter icon and pf-icon not supporting that icon given font awesome v5 */
+      if (this.icon === 'twitter') {
+        newDiv.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" fill="currentcolor" height="24" width="24" viewBox="0 0 512 512"><!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M389.2 48h70.6L305.6 224.2 487 464H345L233.7 318.6 106.5 464H35.8L200.7 275.5 26.8 48H172.4L272.9 180.9 389.2 48zM364.4 421.8h39.1L151.1 88h-42L364.4 421.8z"/></svg><span hidden>${newDiv.innerHTML}</span>`;
```

Comment:
span hidden is there otherwise text shows up next to the svg, pf-icon has the ${newDiv.innerHTML} added as textContent to pf-icon 's fallback slot.  In this case there is no fallback to the SVG but we need to still be able to grab that textContent for usage in the aria-label code.